dali/operators/generic/slice/slice_attr.h

```diff
@@ -122,14 +222,17 @@ class SliceAttr {
           axes = GetDimIndices(shape_layout, axis_names_).to_vector();
         }
 
+        constexpr double i64min = std::numeric_limits<int64_t>::min();
```

Comment:
```suggestion
        constexpr double i64min = static_cast<double>(std::numeric_limits<int64_t>::min());
```
...or you'll fall victim to clang's warnings.